src/components/images/header-logo-lg.tsx

```diff
@@ -10,6 +12,11 @@ type Props = HtmlHTMLAttributes<SVGSVGElement> & {
 
 const HeaderLogoLg = ({height = 89, width = 579, ...props}: Props) => {
   const hasPageBanner = usePageHasTopBanner()
+  const { width: windowWidth } = useWindowSize()
+  
+  if (!height || height === 89) {
+    height = windowWidth >= 1500 ? 126 : windowWidth <= 1500 ? 104 : 72;
```

Comment:
Is it possible to remove the height/width attributes and use classes?

Reply:
@pookmish we can definitely use classes instead and set the width/height attributes to 100%! I'll tweak the conditionals and switch us over to using TW classes 🫡